crates/astria-sequencer/src/transaction/action.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
+use astria_proto::sequencer::v1::{
+    action::Value::{
+        SequenceAction as ProtoSequenceAction,
+        TransferAction as ProtoTransferAction,
+    },
+    Action as ProtoAction,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+use crate::{
+    accounts::TransferAction,
```

Comment:
Clarifying Q: will there be more actions on the accounts module?

Reply:
not currently for in the near future, theoretically if we implement protocol-level account abstraction or native tokens there could be others